app/assets/stylesheets/common.scss

```diff
@@ -516,13 +515,13 @@ body.small-nav {
 }
 
 @mixin dark-map-color-scheme {
-  .leaflet-tile-container,
+  .leaflet-layer,
   .mapkey-table-entry td:first-child > * {
-    filter: var(--dark-mode-map-filter);
+    filter: var(--dark-mode-map-filter, none);
   }
 
-  .leaflet-tile-container .leaflet-tile {
-    filter: none;
+  .no-filter, .dark {
+    --dark-mode-map-filter: none;
```

Comment:
This all seems very confused - you've removed the default value for the filter variable, and set a default of `none` when using it, so why do we need to explicitly set it to none here?